docs/gettingstarted/index.rst

```diff
@@ -168,14 +168,16 @@ Getting started with Flocker
 
          Simulate a Flocker cluster with virtual machines on your laptop (requires `Vagrant <http://www.vagrantup.com/downloads>`_, `VirtualBox <https://www.virtualbox.org/wiki/Downloads>`_):
 
-         .. code-block:: console
+         :version-download:`nodes/Vagrantfile.template`
+
+         .. version-code-block:: console
 
-            you@laptop:~$ git clone \
-              https://github.com/clusterhq/vagrant-flocker && \
-              cd vagrant-flocker && \
-              vagrant up && \
-              [ -e "${SSH_AUTH_SOCK}" ] || eval $(ssh-agent) && \
-              ssh-add ~/.vagrant.d/insecure_private_key
+            mkdir vagrant-flocker
+            cd vagrant-flocker
+            wget https://docs.clusterhq.com/en/|latest-installable|/gettingstarted/nodes/Vagrantfile
+            vagrant up
+            [ -e "${SSH_AUTH_SOCK}" ] || eval $(ssh-agent)
```

Comment:
The previous scripts use `&&` to join the commands intentionally.  That pattern stops the command sequence on the first failure.